maps/tether/tether-09-solars.dmm

```diff
@@ -381,32 +378,31 @@
 /turf/simulated/floor/virgo3b_indoors,
 /area/tether/outpost/solars_shed)
 "aM" = (
-/obj/machinery/power/smes/buildable{
-	charge = 0;
-	output_attempt = 0;
-	outputting = 0;
-	RCon_tag = "Solar Farm - SMES 1"
+/obj/structure/cable/heavyduty{
+	icon_state = "4-8"
 	},
-/obj/structure/cable/heavyduty,
-/turf/simulated/floor/virgo3b_indoors,
-/area/tether/outpost/solars_shed)
+/obj/structure/railing,
+/obj/effect/decal/cleanable/dirt,
+/turf/simulated/floor/virgo3b,
+/area/tether/outpost/solars_outside)
 "aN" = (
-/obj/machinery/power/smes/buildable{
-	charge = 0;
-	output_attempt = 0;
-	outputting = 0;
-	RCon_tag = "Solar Farm - SMES 2"
+/obj/structure/cable/heavyduty{
+	icon_state = "1-2"
+	},
+/obj/machinery/light/small{
+	dir = 8;
+	pixel_x = 0
 	},
-/obj/structure/cable/heavyduty,
-/turf/simulated/floor/virgo3b_indoors,
-/area/tether/outpost/solars_shed)
-"aO" = (
 /obj/effect/decal/cleanable/dirt,
+/turf/simulated/floor/virgo3b,
+/area/tether/outpost/solars_outside)
+"aO" = (
 /obj/machinery/power/smes/buildable{
 	charge = 0;
+	cur_coils = 3;
 	output_attempt = 0;
 	outputting = 0;
-	RCon_tag = "Solar Farm - SMES 3"
+	RCon_tag = "Solar Farm - SMES 1"
```

Comment:
Would numbering like that make any sense when there's only one to begin with?

Reply:
No, I just forgot to change it.